merkle_drop/server.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask
+from flask import jsonify, abort
+from typing import Dict
+from .airdrop import get_item, to_items, get_balance
+from .merkle_tree import create_proof, build_tree
+from eth_utils import encode_hex, is_checksum_address, to_canonical_address
+
+app = Flask("Merkle Airdrop Backend Server")
+airdrop_dict = None
+airdrop_tree = None
+
+
+@app.errorhandler(404)
+def not_found(e):
+    return jsonify(error=404, message="Not found"), 404
+
+
+@app.errorhandler(400)
+def bad_request(e):
+    return jsonify(error=400, message=e.description), 400
+
+
+@app.errorhandler(500)
+def internal_server_error(e):
+    return jsonify(error=500, message="There was an internal server error"), 500
+
+
+@app.route("/airdrop/<string:address>", methods=["GET"])
```

Comment:
It is not really clear what will be returned. 
Maybe `/entitlement/addresses/<string:address>`?

Reply:
Okay `airdrop` might be some kind of meta-word. I'm happier with `entitlement`, but not with added `addresses` as it is not RESTful.

Reply:
Okay, I am also fine without `addresses`